anaconda_project/project.py

```diff
@@ -1573,3 +1585,18 @@ def use_changes_without_saving(self):
         """
         self.project_file.use_changes_without_saving()
         self.lock_file.use_changes_without_saving()
+
+    @property
+    def bootstrap_env_prefix(self):
+        """Fullpath to bootstrap environment prefix."""
+        return join(self._directory_path, 'envs', 'bootstrap-env')
+
+    @property
+    def is_running_in_bootstrap_env(self):
```

Comment:
This does **not** feel like a property, this is answering a question... like a function.

Reply:
mmmm.... kind of disagree.. A property __is__ a function... just a special one. This one happens to behave like a flag. There is not rule that says properties cannot behave like status checks variables/functions, right?

Reply:
>A property is a function... just a special one. 

I know the internals, I am talking about how properties are used, they are basically the state of something so naming them in ways that they feel like a function is what I am arguing here.

```

class Fabio():

    @property
    def age(self):
        return datetime.now() - self.date_of_birth

    def mood(self):
        return self._mood

    def is_angry(self):
        return self._mood == self.MOOD_ANGRY
 ```

Reply:
meh... yeah, I get your point. I just feel like it's more bikeshedding than really adding value. I can do the change anyway :)

Reply:
You are the boss ;-) I make the nitpicks, you make the calls

Reply:
But we do need the rebase/merge with master though!

Reply:
Yup.. Will do it but let's leave it out of the dev build today